llvm/test/CodeGen/NVPTX/lower-alloca.ll

```diff
@@ -17,7 +17,22 @@ define void @kernel() {
   ret void
 }
 
+define void @alloca_in_explicit_local_as() {
+; LABEL: @lower_alloca_addrspace5
+; PTX-LABEL: .visible .func alloca_in_explicit_local_as(
+  %A = alloca i32, addrspace(5)
+; CHECK-NOT: addrspacecast ptr %A to ptr addrspace(5)
```

Comment:
I'd prefer to see a positive check here matching what we *do* want to see.
Negative checks are rather fragile. E.g. an unintentional typo in what we match will makes them always succeed.

Reply:
makes sense, adding a few positive checks to just to check the effect of `-nvptx-lower-alloca `